changelogs/current.yaml

```diff
@@ -153,7 +153,7 @@ new_features:
     added :ref:`cluster_header <envoy_v3_api_field_config.route.v3.RouteAction.RequestMirrorPolicy.cluster_header>` in :ref:`request_mirror_policies <envoy_v3_api_field_config.route.v3.RouteAction.request_mirror_policies>` to allow routing shadow request to the cluster specified in the request_header.
 - area: dubbo_proxy
   change: |
-    add metadata_match in :ref:`RouteConfiguration <envoy_v3_api_field_extensions.filters.network.dubbo_proxy.v3.RouteAction>` support to the dubbo proxy.
+    add metadata_match in :ref:`RouteConfiguration <envoy_v3_api_field_extensions.filters.network.dubbo_proxy.v3.RouteAction.metadata_match>` support to the dubbo proxy.
```

Comment:
add :ref:`metadata_match <envoy_v3_api_field_extensions.filters.network.dubbo_proxy.v3.RouteAction.metadata_match>` support to the dubbo proxy.